db/schema.rb

```diff
@@ -10,20 +10,42 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.1].define(version: 2024_03_22_145705) do
+ActiveRecord::Schema[7.1].define(version: 2024_03_25_095831) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "pg_trgm"
   enable_extension "plpgsql"
 
   # Custom types defined in this database.
   # Note that some types may not work with other database engines. Be careful if changing database.
-  create_enum "claim_status", ["internal", "draft", "submitted"]
+  create_enum "claim_status", ["internal", "draft", "submitted", "discarded"]
```

Comment:
Is this supposed to be here? It looks like a hangover from #399 which originally [introduced a new enum value](https://github.com/DFE-Digital/itt-mentor-services/pull/399#pullrequestreview-1954335918), but subsequently reverted the change.

Reply:
Oh, old migration. Good catch